### ISSUE TITLE
Allow non mandatory app updates as opt-in

### ISSUE THREAD

**Issue by CMTRACE** [dappnode/DAppNode#359]
I'd like to see app updates that are non-critical or non-mandatory listed as opt-in

Instead of presenting the critical app update and firing a notification, non-critical apps should be presented in the Dapp store

Example:

Prysm 1.4.4 has been posted for over a week but hasn't been presented in the dappnode store. 

I understand it's not mandatory but it would be nice to keep on the absolute latest bits from the implementation teams. 

Alternatively, the team could expand their review scope to include non cortical updates so DappNode users are able to be on the latest bits

**Comment by dapplion** [comment]
@CMTRACE thanks for the proposal! Agree with you but the best solution would be to have a faster review period of new Prysm updates. As part of our due process we install, test and review every single update from Prysm before releasing to DAppNode users to be sure it won't break our setup. But since it's a manual process it can be delayed.

If you want to stay on top and contribute, a bot prepares new Prysm package versions a few hours after Prysm releases. See this PR for example https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/11. What you could do is
- subscribe to notifications of that repo.
- when a new PR is created by the bot install the new version (now you have access to the latest versions immediately). Please understand its an untested version that may break your setup!
- if it installs and runs correctly please leave a comment and approve the PR stating its safe for you. :heart:

**Comment by CMTRACE** [comment]
I appreciate you're reply and I do understand the content of this update reduces the urgency somewhat.

If someone was to install this what sort of breakages would you expect? (worse case)

I would be totally fine with a stopped client or shredded DB if there's some support to fix (I trust there  would be)

But is there any risk in DappNode starting two clients with the same keys?, forgive my ignorance I'm not sure how the update process is handled internally.

Basically what I'm saying is, if the worst thing happening is a stopped client and some troubleshooting with you guys on discord, that's actually fine for my risk appetite.

If however, there's a risk in being slashed that's a great deal higher than my risk appetite 😅

**Comment by dapplion** [comment]
> I appreciate your reply and I do understand the content of this update reduces the urgency somewhat.
> 
> If someone was to install this what sort of breakages would you "expect"? (worse case)
> 
> I would be totally fine with a stopped client or shredded DB if there's some support to fix (I trust there would be)
> 
> But is there any risk in DappNode starting two clients with the same keys?, forgive my ignorance I'm not sure how the update process is handled internally.
> 
> Basically what I'm saying is, if the worst thing happening is a stopped client and some troubleshooting with you guys on discord, that's actually fine for my risk appetite.
> 
> If however, there's a risk in being slashed that's a great deal higher than my risk appetite sweat_smile

I would say the most likely scenario is a shred db or the binary just crashing in a loop. However, we can't promise support in a timely manner in that event, we are all in EU timezone. The most cautious approach would be to test the update on Prater first and then mainnet. That would reduce the change of a crash very significantly
